moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -149,6 +182,74 @@ const ComponentForm = ({
             minRows={4}
           />
         </Grid>
+        <Grid item xs={12}>
+          <FormControlLabel
+            control={
+              <Switch
+                checked={useComponentPhase}
+                onChange={() => setUseComponentPhase(!useComponentPhase)}
+                name="useComponentPhase"
+                color="primary"
+              />
+            }
+            labelPlacement="start"
+            label="Use component phase"
+            style={{ color: "gray", marginLeft: 0 }}
+          />
+          {useComponentPhase && (
+            <FormHelperText>
+              Assign a phase to the component to differentiate it from the
+              overall phase of this project
+            </FormHelperText>
+          )}
+        </Grid>
+        {useComponentPhase && (
+          <>
+            <Grid item xs={12}>
+              <ControlledAutocomplete
+                id="phase"
+                label="Phase"
+                options={areOptionsLoading ? [] : phaseOptions}
```

Comment:
I think this ternary can go because the `usePhaseOptions` hook return `[]` if it hasn't prepared the options yet.

Reply:
thanks for catching that - will cut that out!